### ISSUE TITLE
Keyboard on Android Nougat not working after receiving in app message

### ISSUE THREAD

**Issue by cwelk** [Appboy/appboy-cordova-sdk#15]
Hi,

On **Android 7.1.x** if an appboy in app message is closed, the keyboard does not open on input fields focus anymore. _(If the no appboy message is received the keyboard opens on focus)_

We tested on Android 4.x and 5.x also, where the issue was not reproducible.

To reproduce:

1. download sample app https://www.dropbox.com/s/vu9opo3ulzt71fg/myapp.zip?dl=0 _(ionic starter + appboy plugin)_

2. Switch to "**Chats**" tab _-> in app message appears_

3. Switch to "**Account**" tab and tap on one of the inputs _-> no keyboard opens_  

Our setup is:

```
cordova 6.5.0
cordova-android 6.1.2
android:minSdkVersion="16" 
android:targetSdkVersion="25"
ionic 1.3.2
```

To run the test project:

```
npm install -g ionic
cordova platform add android
ionic run android
``` 
(with android attached to usb)

**Comment by radixdev** [comment]
Hey @cwelk,

I've reproduced the keyboard issue you're seeing. This appears to be caused with how we display in-app messages in the core Appboy Android SDK and not the Cordova SDK itself. Unfortunately, this means that a fix won't be available as quickly. 

I will let you know when we have a fix released and released into the main Cordova plugin.

Thanks,
Julian

**Comment by cwelk** [comment]
Great thank you!

**Comment by yamov** [comment]
Hey, @radixdev. Any updates on this issue?

**Comment by radixdev** [comment]
Hey @yamov / @cwelk ,

We're testing the fix in the core Appboy Android SDK that fixes this issue. We have a planned monthly release for the core SDK. Once the fix is live, it will either be immediately available via maven or will require a simple Appboy Cordova SDK update. In either case, I will ping this issue when it goes live. Thank you for your patience in getting this fixed and thank you for alerting us!

Thanks,
Julian

**Comment by yamov** [comment]
Hey @radixdev. Thanks, glad to hear that :).

**Comment by radixdev** [comment]
Hey @cwelk / @yamov ,

We just released Appboy Android SDK 2.0.5, which contains a fix for the keyboard issue, as re-tested with this sample app. Thanks to how the Appboy Cordova plugin uses [gradle versioning](https://github.com/Appboy/appboy-cordova-sdk/blob/master/plugin/src/android/build-extras.gradle#L7), this fix should be pulled in automatically, without the need of a Appboy Cordova update on your part.

Let me know if this fixes your issue,
Julian

**Comment by yamov** [comment]
Hey @radixdev. The problem seems to be solved. The issue can be closed now.
Thanks ;)
